libs/api/v1/src/lib/api-v1.ts

```diff
@@ -35,7 +35,7 @@ export const apiV1 = new Elysia({ prefix: '/v1' })
     const errorCode: ApiErrorCode | undefined = await validateApiKey(headers.authorization)
     if (errorCode) {
       set.headers['Content-Type'] = 'application/json; charset=utf8'
-      return errorFn(ERROR_LIST[errorCode].code, formatApiErrorResponse(errorCode))
+      throw errorFn(ERROR_LIST[errorCode].code, formatApiErrorResponse(errorCode))
```

Comment:
**question:** Where does throwing instead of returning come from? Didn't it work that way?